alembic/README.md

```diff
@@ -0,0 +1,29 @@
+# Alembic Database Migration tool
+
+## Getting Started
+
+To be able to perform migrations, you will need to set some Environment Variables for the tool to connect to your DB. 
+The following variables are required:
+```
+POSTGRES_USERNAME={database user}
+POSTGRES_PASSWORD={database password}
+POSTGRES_HOST={database address}
+POSTGRES_NAME={database name}
```

Comment:
could we make it easier and use something like https://pypi.org/project/python-dotenv/ 
and avoid the need to set these on the command line (as that is something I will likely forget -- regularly)

Reply:
This is already being used under the hood so have updated the readme to be explicit in saying you can add them there.